gramps/plugins/importer/importxml.py

```diff
@@ -156,6 +157,17 @@
 INSTANTIATED = 1
 
 
+def to_bool(value):
+    """
+    Utility function to convert attr value to boolean.
+    """
+    if isinstance(value, str):
+        value = value.lower()
+    if value in (False, "false", "f", "no", "n", "off", "0", 0):
+        return False
+    return True
```

Comment:
Is it wise to assume that any value not `False` is `True`?
`to_bool("Doug")` would return True. In the case of private, I agree it defaults safe by doing so. However if future someone wants to use `to_bool` in a different context, they may prefer it to defaulted to False.
I'd rather it threw an error if the data is not clearly one of the false or true values.

Reply:
In regular Python, anything other than empty sequences (`""`, `[]`, `{}`), `False`, `0`, and `None` will return `True`:

```
if "Steven":
    print("True!")
```

`if` (hardly) ever would raise an exception. (Although numpy does). 

I'd rather parse it into `True` or `False`, rather than crashing. I didn't put the function in a general module, as it is specific to this use.

Reply:
Refined `to_bool()` and added more docs.